code/modules/mob/living/ventcrawling.dm

```diff
@@ -53,17 +53,19 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/components/unary
 			if(!client)
 				return
 
-			if(iscarbon(src) && ventcrawler < 2)//It must have atleast been 1 to get this far
-				var/failed = 0
-				var/list/items_list = get_equipped_items()
-				if(items_list.len)
-					failed = 1
-				for(var/obj/item/I in held_items)
-					failed = 1
-					break
-				if(failed)
-					src << "<span class='warning'>You can't crawl around in the ventilation ducts with items!</span>"
-					return
+			if(iscarbon(src) && contents.len && ventcrawler < 2)//It must have atleast been 1 to get this far
```

Comment:
I don't see why this would require a change, mind explaining why this entire check was altered?